editor/editor_dock_manager.cpp

```diff
@@ -392,7 +393,10 @@ void EditorDockManager::_dock_move_selected_to_bottom() {
 	dock->call("_set_dock_horizontal", true);
 
 	bottom_docks.push_back(dock);
-	EditorNode::get_bottom_panel()->add_item(dock->get_name(), dock, true);
+	// Force FileSystem dock to appear first in the list, and give it a shortcut.
+	// Since only the FileSystem dock can currently be moved to the bottom,
+	// the shortcut is assumed to be for the FileSystem dock.
+	EditorNode::get_bottom_panel()->add_item(dock->get_name(), dock, ED_SHORTCUT_AND_COMMAND("bottom_panels/toggle_filesystem_bottom_panel", TTR("Toggle FileSystem Bottom Panel"), KeyModifierMask::ALT | Key::F), true);
```

Comment:
We shouldn't make such assumptions. You could either check if `dock` is FileSystemDock or add a method to get bottom panel shortcut (can be called with `call()` like the one above).

Reply:
Also, this shortcut can't be added here or in load since it can be called multiple times, leading to errors showing `The Command 'Toggle FileSystem Bottom Panel' already exists. Unable to add it.`
It should probably be in `add_control_to_dock` or removed for now.

Reply:
I've pushed a fix for this as a second commit, which allows docks to define toggle shortcuts. These currently only work when the dock is moved to a bottom panel, but we could allow them to be used to hide/show docks in the future.